internal_tools/redirection/main.tf

```
@@ -0,0 +1,25 @@
+data "terraform_remote_state" "domain" {
+
+  backend = "s3"
+  config = {
+    bucket = "terraform-state-xain"
+    key    = "domains/terraform.tfstate"
+    region = "eu-central-1"
+  }
+}
+
+module "redirection" {
+  source = "github.com/xaynetwork/xayn_terraform_modules//internal/redirection?ref=v0.5.0"
+
+  domain_name    = "support.xayn.com"
+  hosted_zone_id = data.terraform_remote_state.domain.outputs.xayn_primary_hosted_zone_id
+  host_name      = "xainag.atlassian.net/servicedesk/customer/portals"
+
+  # Optional
+  tags = {
+    Environment = "production"
+    User        = "provider"
```

```suggestion
    Environment = "production"
    User        = "provider"
```
Do we need any of this?